src/Components/Common/Button.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+interface ButtonProps {
+  children?: React.ReactElement | string
+  className?: string
+}
```

Comment:
```suggestion
interface ButtonProps {
  children: React.ReactElement | string
  className?: string
}
```
`children`은 optional이 되면 어떤 버튼이 되나요?

Reply:
요 버튼은 스트링이 없는 버튼까지 커버치는 버튼입니다!! 혹시 더 좋은 방법이 있을까용?